src/editors/framework/site/base-site-information.php

```diff
@@ -80,6 +80,7 @@ public function get_site_information(): array {
 			'isRtl'                 => \is_rtl(),
 			'isPremium'             => $this->product_helper->is_premium(),
 			'siteIconUrl'           => \get_site_icon_url(),
+			'isPrivateBlog'         => ! \get_option( 'blog_public' ),
```

Comment:
Is there a reason we turned the `( (string) get_option( 'blog_public' ) ) === '0'` check into the more generic `! \get_option( 'blog_public' )` one?

If not, I'd rather we kept the same check here

Reply:
According to [get_option documentation](https://developer.wordpress.org/reference/functions/get_option/), in case the option was never saved we will get `false`, if it is saved we are getting a boolean that is then returned as either empty string or "1". I see no reason to convert it back to string to then check it it equals to a "0" string and return a boolean value that will then be interpreted as "1" or empty string.

Reply:
Update:
`( (string) get_option( 'blog_public' ) ) === '0'` is actually a bug.